src/pages/p2p/components/_dp2p.js

```diff
@@ -91,6 +101,15 @@ const Row = styled.div`
         margin-top: 40px;
     }
 `
+
+const StyledIFrame = styled.iframe`
+    width: 560px;
+    height: 315px;
+
+    @media ${device.mobileM} {
+        width: 310px;
+    }
+`
```

Comment:
It's still gonna break from 560px width to mobileM.

Reply:
So what is the better width for use here?

Reply:
Better to set the breakpoint for like tablet/tabletS and the width to say like 90% with a max-width for it.

Reply:
okay sure, let me work on it